examples/basic/src/server.js

```diff
@@ -0,0 +1,38 @@
+require("@babel/register")({
+  extensions: [".ts", ".tsx", ".js", ".jsx"],
+})
+
+const express = require("express")
+const webpack = require("webpack")
+const webpackDevMiddleware = require("webpack-dev-middleware")
+const webpackConfig = require("../webpack.config")
+const compiler = webpack(webpackConfig)
+
+const app = express()
+
+app.use(
+  webpackDevMiddleware(compiler, {
+    publicPath: webpackConfig.output.publicPath,
+    serverSideRender: true,
+    stats: "errors-only",
+  })
+)
+
+app.get("/", (req, res) => {
+  res.send(`
+    <html>
+      <head>
+        <title>@artsy/fresnel - SSR Example</title>
```

Comment:
```suggestion
        <title>@artsy/fresnel - CSR Example</title>
```

Reply:
```suggestion
        <title>@artsy/fresnel - Basic Example</title>
```

We've called it "Basic" throughout the docs; also, it took me a bit to figure out what "CSR" meant.

Reply:
haha what _does_ csr mean?

Reply:
Hah. Yeah, good call